docs/recipes/hydrating-state.md

```diff
@@ -0,0 +1,100 @@
+# Hydrating State
+
+Often times your application will need to start in a specific
+state. For example: server rendered pages that need to be hydrated in
+a specific state, or applications that save data in `localStorage` for
+return visits.
+
+Microcosm ships with several utilities that make this easy.
+
+## Resetting state
+
+The `reset` method wipes away existing state, replacing it with new
+data. Where domain keys are unspecified, values are replaced with the
```

Comment:
Took another pass:

> The `reset` method wipes away existing state, replacing it with new
data. If this data object does not include a key managed by a domain,
it will reset that value to the initial state of the domain.